internal/bminventory/inventory.go

```diff
@@ -603,7 +603,7 @@ func (b *bareMetalInventory) RegisterClusterInternal(
 	}
 	setPullSecret(&cluster, ps)
 
-	if err = validations.ValidateClusterNameFormat(swag.StringValue(params.NewClusterParams.Name)); err != nil {
+	if err = validations.ValidateClusterNameFormat(swag.StringValue(params.NewClusterParams.Name), *params.NewClusterParams.Platform.Type); err != nil {
```

Comment:
iinm, it's nil-safe here, but would be cleaner to use 'getPlatformType' func: https://github.com/openshift/assisted-service/blob/be2c5764fd0f2d90750d650aff267cda579fa4ef/internal/bminventory/inventory.go#L1818

Reply:
Gotcha, modified to use this function, thanks!